README.rst

```diff
@@ -14,103 +14,69 @@ Microsoft Azure SDK for Python
     :target: https://ci.appveyor.com/project/lmazuel/azure-sdk-for-python/branch/master
 
 This project provides a set of Python packages that make it easy to
-access the Microsoft Azure components such as ServiceManagement, Storage\*, and ServiceBus.
+access Management of Microsoft Azure components (Virtual Machines, ...) or Runtime (ServiceBus using HTTP, Batch, Monitor).
+Complete feature list of this repo and where to find Python packages not in this repo can be found on our 
+`Azure SDK for Python features chapter on ReadTheDocs <http://azure-sdk-for-python.readthedocs.io/en/latest/index.html#features>`__.
 
 The SDK supports Python 2.7, 3.3, 3.4, 3.5 and 3.6.
 
-\*Looking for the Azure Storage client library?  It moved to a `new GitHub repository <https://github.com/Azure/azure-storage-python>`__.
-
-See important information if you're currently using this SDK < 1.0 in `this issue <https://github.com/Azure/azure-sdk-for-python/issues/440>`__.
+See important information if you're currently using the ``azure`` package < 1.0 in `this issue <https://github.com/Azure/azure-sdk-for-python/issues/440>`__.
 
 
 INSTALLATION
 ============
 
-**The latest recommended release is currently a release candidate, tell this to pip to install it!**
+You can install individually each library for each Azure service:
 
-The `azure` bundle meta-package will install all Azure SDKs at once:
+.. code-block:: console
 
-- Use the ``--pre`` flag: ``pip install --pre azure``
+   $ pip install azure-batch         # Install the latest Batch runtime library
+   $ pip install azure-mgmt-storage  # Install the latest Storage management library
 
-- Specify the version:  ``pip install azure==2.0.0rc6``
+Preview packages can be installed using the ``--pre`` flag:
 
-You can also install only what you exactly need:
+.. code-block:: console
 
-- Use the ``--pre`` flag: ``pip install --pre azure-mgmt-compute``
+   $ pip install --pre azure-mgmt-trafficmanager # will install the latest Traffic Manager
 
-- Specify the version:  ``pip install azure-mgmt-compute==0.30.0rc6``
 
-If you want to install ``azure`` from source::
+You can also install a set of Azure libraries in a single line using the ``azure`` meta-package.
+Since not all packages in this meta-package are
+published as stable yet, the ``azure`` meta-package is still in release candidate.
+However, the core packages, from code quality/completeness perspectives can at this time be considered "stable" 
+- it will be officially labeled as such in sync with other languages as soon as possible. 
```

Comment:
"the ``azure`` package will be officially labeled stable in sync with other language SDKs as soon as possible"